HACKING.md

```diff
@@ -65,49 +65,65 @@ Run the unit tests with:
 
     $ hack/test-go.sh
 
-or an individual package unit test with:
+or an individual package with:
 
-    $ hack/test-go.sh pkg/build
+    $ hack/test-go.sh relative/path/to/package
 
 or an individual package and all packages nested under it:
 
-    $ hack/test-go.sh pkg/build/...
+    $ hack/test-go.sh relative/path/to/package/...
 
 To run only a certain regex of tests in a package, use:
 
-    $ hack/test-go.sh pkg/build -test.run=SynchronizeBuildRunning
+    $ GOTEST_FLAGS='-test.run=SpecificTestName' hack/test-go.sh relative/path/to/package
 
-To get verbose output add `-v` to the end:
+To get verbose output for the above example:
 
-    $ hack/test-go.sh pkg/build -test.run=SynchronizeBuildRunning -v
+    $ VERBOSE=true GOTEST_FLAGS='-test.run=SpecificTestName' hack/test-go.sh relative/path/to/package
```

Comment:
Why not `GOTEST_FLAGS='-run=SpecificTestName -v'`?
We shouldn't need to learn a new set of env vars when standard `go` and `go test` flags would do the trick, wdyt?